File: Tutorials/wk 6/Lloyd.java
import java.util.Scanner;
import java.util.Random;
public class Lloyd{
    public static void main(String[] args) {
        Random rand = new Random(51);
        int number = rand.nextInt();
        //System.out.println(number);
        int number_of_guesses = 0;
        int number_of_chances = 3;

        while (number_of_guesses < number_of_chances){




           Scanner scanner = new Scanner(System.in);
            System.out.println("Guess a number from 1 to 50: ");
            int result = scanner.nextInt();
            number_of_guesses = number_of_guesses + 1;
            if(result == number){
                System.out.println("YOU GOT THE RIGHT NUMBER. WHOO HOO!");
                break;
            }else if(result > number){
                System.out.println("Your guess is too high");
            }else if(result > number){
                System.out.println("Your guess is too low");
            }

        }    
     }
}

File: Tutorials/wk 6/Lloyd.py
import random as rd
number = rd.randint(1, 50)
number_of_guesses = 0
number_of_chances = 3
while number_of_guesses < number_of_chances:
    print("Guess a number between 1 to 50: ")
    guess = input()
    guess = int(guess)
    number_of_guesses = number_of_guesses + 1
    
    if guess < number:
        print("You're guess is to low.")
    elif guess > number:
        print("You're guess is too high. ")
    elif guess == number:
        print("WHOO! that's the number.")

    elif number_of_guesses == 0:
        print("You ran out of guesses the correct number was ",number)
    

    




